FND/Accounting/ISO4217-CurrencyCodes.rdf

```diff
@@ -1680,8 +1675,10 @@
 		<rdf:type rdf:resource="&fibo-fnd-acc-cur;Currency"/>
 		<rdfs:label>Kuna</rdfs:label>
 		<skos:definition xml:lang="en">the currency Kuna</skos:definition>
+		<skos:note>Effective 1 Jan 2023, Croatia will use the Euro as its primary currency. The Kuna (HRK) and Euro (EUR) will be used during the parallel circulation period from 1 January 2023 to 14 January 2023 inclusive. The period of mandatory dual price display lasts from 5 September 2022 to 31 December 2023. As of 1 January 2023, the Kuna should be listed as the old/historic currency of Croatia. The exchange rate is fixed at EUR 1 = HRK 7.53450</skos:note>
```

Comment:
should we indicate the status e.g. using owl:deprecated?

Reply:
I thought about that ... maybe, and then it would be removed altogether in 2024?  I'll add the deprecation for it and its code.